filesystems/samba2/README.md

```diff
@@ -0,0 +1,14 @@
+# Introduction
+
+Samba is the standard Windows interoperability suite of programs for Linux and Unix.
+
+## Run Samba Docker Compose File.
+docker-compose up --build
```

Comment:
We automatically build and deploy the image to the private nexus, so if you're logged in, there's no need to build the image.

Reply:
Yes, but I put it here for developers trying this image locally. Not for you, how to build the image 😉